modules/gltf/gltf_document.cpp

```diff
@@ -3519,17 +3524,21 @@ Error GLTFDocument::_parse_materials(Ref<GLTFState> state) {
 				material->set_cull_mode(BaseMaterial3D::CULL_DISABLED);
 			}
 		}
-
 		if (d.has("alphaMode")) {
 			const String &am = d["alphaMode"];
+			material->set_depth_draw_mode(BaseMaterial3D::DEPTH_DRAW_ALWAYS);
 			if (am == "BLEND") {
-				material->set_transparency(BaseMaterial3D::TRANSPARENCY_ALPHA_DEPTH_PRE_PASS);
+				material->set_transparency(BaseMaterial3D::TRANSPARENCY_ALPHA);
 			} else if (am == "MASK") {
+				float alpha_cutoff = 0.5f;
 				material->set_transparency(BaseMaterial3D::TRANSPARENCY_ALPHA_SCISSOR);
+				material->set_alpha_antialiasing(BaseMaterial3D::ALPHA_ANTIALIASING_ALPHA_TO_COVERAGE_AND_TO_ONE);
```

Comment:
I'm not sure if this is the best approach. Won't this only work when MSAA is enabled? E.g. when alpha to coverage is available?

Reply:
According to the source designs, it works when aa is off.

https://github.com/godotengine/godot/pull/40364

https://bgolus.medium.com/anti-aliased-alpha-test-the-esoteric-alpha-to-coverage-8b177335ae4f

\<Ben Golus> In fact if MSAA is disabled Alpha to Coverage falls back to alpha test. This means it isn’t only useful for VR or other games where you’re guaranteed to have MSAA enabled. Basically there’s rarely a reason to use straight alpha testing ever!